client/me/security-2fa-key/name.jsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import FormButton from 'components/forms/form-button';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+
+class Security2faKeyAddName extends React.Component {
+	static propTypes = {
+		onNameSubmit: PropTypes.func.isRequired,
+		onCancel: PropTypes.func.isRequired,
+	};
+
+	state = {
+		error: false,
+		keyName: '',
+	};
+
+	componentDidMount = () => {
+		this.setState( { error: false, keyName: '' }, () => {
```

Comment:
Do we really need the `setState` call in `componentDidMount`? The state is initialized to the same values in the statement above.

Reply:
I'd say it's not needed.

Reply:
Kill it with 🔥